app/controllers/ViewMovementController.scala

```diff
@@ -73,17 +74,18 @@ class ViewMovementController @Inject()(mcc: MessagesControllerComponents,
     authorisedDataRequestAsync(exciseRegistrationNumber, viewMovementBetaGuard(exciseRegistrationNumber, arc)) { implicit request =>
       getMovementService.getLatestMovementForLoggedInUser(exciseRegistrationNumber, arc).flatMap { movement =>
         helper.movementCard(currentSubNavigationTab, movement).map { movementCard =>
+
           Ok(view(
             ern = exciseRegistrationNumber,
             arc = arc,
-            isConsignor = movement.consignorTrader.traderExciseNumber.fold(false)(_ == exciseRegistrationNumber),
             subNavigationTabs = SubNavigationTab.values,
             currentSubNavigationTab = currentSubNavigationTab,
             movementTabBody = movementCard,
             historyEvents = movement.eventHistorySummary
               .map(timelineHelper.timeline(_))
               .getOrElse(Seq.empty[TimelineEvent]),
-            messageStatistics = request.messageStatistics
+            messageStatistics = request.messageStatistics,
+            actionLinksBody = actionHelper.movementActions(movement)
```

Comment:
Rather than pass the HTML to the view to render at `Controller` level, it might be better to inject the helper within the view then call the helper from the view itself?

Same with maybe refactoring the `movementTab` body as well? Again, so that HTML isn't passed from a `Controller` to a `View` (although that could be done as a separate ticket).

Reply:
This is more of a personal opinion though, as I always think the Controller should only be concerned with handling which view to render and the saving and routing. Rather than the Controller actively rendering content to then pass to a view. As the rendering of content seems more like the views responsibility, and it can call on the helpers to achieve it.

Reply:
I agree. I followed suite purely because of the `movementTabBody` was done like that so was trying to keep it consistent.

Reply:
Moved the action helper from controller to view https://github.com/hmrc/emcs-tfe-frontend/pull/134/commits/96f2ca11c67fbc7c5ed23734bdfeea0e60191157